src/test/java/com/microsoft/aad/msal4j/AuthorizationRequestUrlParametersTest.java

```diff
@@ -121,5 +127,6 @@ public void testBuilder_optionalParameters() throws UnsupportedEncodingException
         Assert.assertEquals(queryParameters.get("correlation_id"), "corr_id");
         Assert.assertEquals(queryParameters.get("login_hint"), "hint");
         Assert.assertEquals(queryParameters.get("domain_hint"), "domain_hint");
+        Assert.assertEquals(queryParameters.get("claims"), "{\"id_token\":{\"auth_time\":{\"essential\":true}},\"access_token\":{\"auth_time\":{\"essential\":true},\"xms_cc\":{\"values\":[\"llt\",\"ssm\"]}}}");
     }
 }
```

Comment:
client capabilities should be sent to token endpoint as well, here it is send only to authorization endpoint

Reply:
In the latest commit capabilities are added to the parameters of requests to the token endpoint.